cmd/lncli/walletrpc_active.go

```diff
@@ -673,8 +680,11 @@ func fundPsbt(ctx *cli.Context) error {
 		return cli.ShowCommandHelp(ctx, "fund")
 	}
 
+	minConfs := int32(ctx.Uint64("min_confs"))
```

Comment:
just checking that we are ok with if a user sets a negative number, we will be `math.Abs`ing it here?